### PR TITLE
Preparing version 0.2

### PR DESCRIPTION
**Pull request by alix-tz**
Cf. #7 

---

- [x] key for production software (#3)
- [x] add institution name and role (#2)
- [x] ~~add `cust-latk`~~ add a free text qualifier for scripts (#4 and #1)
- [x] add `orcid` (#10)

### CODE REVIEW

File: 2022-04-15/schema.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                            "transcriber", "aligner", "project-manager", 
+                            "support", "digitization", "quality-control"
+                        ]
+                    },
+                    "uniqueItems": true
+                }
+            }
+        }
+    },
+    "production-software": {
```
Comment (PonteIneptique) [inline 2022-04-15/schema.json:165]: Should it be required ? I believe so...

Comment (alix-tz) [reply 2022-04-15/schema.json:165]: Then shouldn't we make `title` required too?

Comment (alix-tz) [reply 2022-04-15/schema.json:170]: Nevermind, I hadn't seen the last item in the schema 😶‍

Comment (alix-tz) [reply 2022-04-15/schema.json:170]: Done!

**Comment by PonteIneptique** [comment]
Merging for preparing the future.

**Comment by PonteIneptique** [comment]
It is!

Le mar. 22 mars 2022 à 9:17 PM, Alix Chagué ***@***.***> a
écrit :

**Pull request merged**